src/other/kabedon/kd78s/kabedon78s.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Kabe Don 78S",
+  "vendorId": "0x4B44",
+  "productId": "0x3738",
+  "lighting": {
+        "extends": "qmk_rgblight",
+        "underglowEffects": [
+            ["All Off", 0],
+            ["Solid Color", 1],
+            ["Solid - Highlighted Mods", 1],
+            ["Vertical Gradient", 1],
+            ["Horizontal Gradient", 1],
+            ["Breathing", 1],
+            ["Colorband - Saturation", 1],
+            ["Colorband - Brightness", 1],
+            ["Pinwheel - Saturation", 1],
+            ["Pinwheel - Brightness", 1],
+            ["Spiral - Saturation", 1],
+            ["Spiral - Brightness", 1],
+            ["Cycle All", 0],
+            ["Cycle Horizontal", 1],
+            ["Cycle Vertical", 1],
+            ["Rainbow Chevron", 1],
+            ["Cycle - In/Out", 1],
+            ["Cycle - In/Out Dual", 1],
+            ["Cycle - Pinwheel", 1],
+            ["Cycle - Spiral", 1],
+            ["Dual Beacon", 1],
+            ["Rainbow Beacon", 1],
+            ["Rainbow Pinwheels", 1],
+            ["Raindrops", 1],
+            ["Jellybean Raindrops", 1]
+        ]
+    },
+  "matrix": { "rows": 6, "cols": 18 },
+  "layouts": {
+    "labels": [
+      "ROWS : F1~F12",
+      "COLS : G1~G5" ,
+      "PAD : 21",
+      "Split : Pad +",
+      "Split : Pad Enter",
+      "Split : Pad 0"
+    ],
```

Comment:
It is not very clear what these do. Perhas they should be "Function Row", "Macro Columns", "Extra Numpad Keys", "Split Numpad +", "Split Numpad Enter", "Split Numpad Zero".